app/dao/services_dao.py

```diff
@@ -284,7 +289,10 @@ def dao_create_service(
     if service_permissions is None:
         service_permissions = DEFAULT_SERVICE_PERMISSIONS
 
-    organisation = dao_get_organisation_by_email_address(user.email_address)
+    if organisation_id:
+        organisation = dao_get_organisation_by_id(organisation_id)
```

Comment:
I suspect we wouldn't come across this case in production, but is there a way we could fail more gracefully if the `org_id` we fetch from the CRM org notes doesn't exist in the DB? Currently we just get a "technical difficulties" screen in this case.
Maybe we can still create the service and not add the org? Adding some logging here or in `service/rest.py` could be helpful as well.

Reply:
Great catch Will! I thought I had tested that case but I am seeing an error when the organisation id is missing from the DB. I have pushed a fix - could you re-test?

Reply:
Re-tested and it looks great! The service gets created without issue when the org doesn't exist yet in the DB. ✅